pages/services/nifi/0.4.0-1.8.0/security/index.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: layout.pug
+navigationTitle: Security
+title: Security
+menuWeight: 49
+excerpt: DC/OS NiFi Service encryption, authentication, and authorization
+featureMaturity:
+enterprise: false
+model: ../../data.yml
+render: mustache
+---
+
+# DC/OS {{model.techName }} Security
+
+The DC/OS {{model.techName }} service supports DC/OS {{model.techName }}’s native transport encryption, authentication, and authorization mechanisms. The service provides automation and orchestration to simplify the usage of these important features.
+
+A good overview of these features can be found  [here](https://{{ model.serviceName }}.apache.org/docs/{{ model.serviceName }}-docs/html/administration-guide.html).
+
+## Transport Encryption and Kerberos Authentication
+With transport encryption enabled, DC/OS {{model.techName }} will automatically deploy all nodes with the correct configuration to encrypt communication via SSL. The nodes will communicate securely between themselves using SSL. SSL authentication requires that all DC/OS {{model.techName }} Nodes present a valid certificate from which their identity can be derived for communicating between themselves.
+DC/OS {{model.techName }} uses the CN of the SSL certificate as the principal for a given Node.
+For example, CN={{ model.serviceName }}-0-node.demonifi, O="Mesosphere, Inc", L=San Francisco, ST=CA, C=US.
+
+The service uses the [DC/OS CA](https://docs.mesosphere.com/latest/security/ent/tls-ssl/) to generate the SSL artifacts that it uses to secure the service. Any client that trusts the DC/OS CA will consider the service’s certificates valid.
+
+<p class="message--note"><strong>NOTE: </strong>Enabling transport encryption is required to use <a href="https://docs.mesosphere.com/1.10/security/ent/tls-ssl/">SSL authentication</a> for authentication, but is optional for Kerberos authentication.</p>
```

Comment:
Is there a reason this links to the 1.10 version and not a later one?

Reply:
not as such... changed to latest